project.clj

```diff
@@ -149,7 +149,7 @@
                                                [puppetlabs/puppetserver ~ps-version]
                                                [puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty9-version]
                                                [org.clojure/tools.nrepl nil]]
-                      :plugins [[puppetlabs/lein-ezbake "1.1.13"]]
```

Comment:
I think this would implicitly remove Java 7 support from Puppet Server 2.x (see https://github.com/puppetlabs/ezbake/blob/master/CHANGELOG.md#120---2017-4-12) /cc @underscorgan

Reply:
ugh I think you're right. I knew that happened somewhere, but didn't find it when skimming the ezbake changelog. I guess I can backport this stuff.

Reply:
In that case I think I'll do the mergeup without the ezbake update in this branch, and add it for 5.3.x, so we can see that go green.